panther_msgs/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package>
+  <name>panther_msgs</name>
+  <version>0.0.0</version>
+  <description>
+    Custom messages for Panther robot
+  </description>
+  <maintainer email="krzysztof.wojciechowski@husarion.com">Krzysztof Wojciechowski</maintainer>
```

Comment:
We discussed about adding support@husarion here. Its TBD probably

Reply:
Done